db/migrate/20191114024115_add_slugs.rb

```diff
@@ -0,0 +1,17 @@
+class AddSlugs < ActiveRecord::Migration[5.2]
+  def change
+    change_table :resource_categories do |t|
+      t.string  :slug
+      t.remove  :short_description,
+                :icon,
+                :seo_title,
+                :seo_description,
+                :seo_keywords,
+                :share_image
+    end
```

Comment:
I added these fields in the very beginning and I don't think they're going to be used anytime soon, so I'm just going to remove them and prevent distractions.